src/pages/Login.vue

```diff
@@ -169,14 +169,13 @@ export default {
           email: 'foo@bar.eu',
           authenticated: true
         }
-
-        // history.push(history.location.state ? history.location.state.from : '/')
       } catch (e) {
         console.error(e)
         this.error = e
         this.loading = false
       } finally {
         this.loading = false
+        this.$router.go(-1)
```

Comment:
How about a redirect to the accounts page?

Reply:
Which page is this?

Reply:
The profile page, which does not exist yet :D

Reply:
This would be a good idea if the user wanted to access the profile page before they logged in. However, consider the following scenario: As a user, I open any page that requires authentication (e.g., `get-help/`). Because I am not logged in, this page will redirect me to the login page. After I have logged in, I expect to be redirected to the previous page.
I assume this to be the default scenario for logging in. And this is what `this.$router.go(-1)` does.